src/components/Article.jsx

```diff
@@ -1,89 +1,96 @@
 import { useParams } from "react-router-dom";
-import { getArticleById } from "../utils/api";
+import { getArticleById, incrementArticleVotes } from "../utils/api";
 import { useContext, useEffect, useState } from "react";
 import popSound from "../assets/popSound.mp3";
 
 import Comments from "./Comments";
 
-
 function Article() {
   const { article_id } = useParams();
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState(false);
   const [article, setArticle] = useState("");
   const [showComments, setShowComments] = useState(false);
-  const [displayedVotes, setDisplayedVotes] = useState(0)
-  const [newVote, setNewVote] = useState(1)
-  
+  const [displayedVotes, setDisplayedVotes] = useState(0);
+  const [newVote, setNewVote] = useState(1);
+
   function popwithcomments() {
     setShowComments(!showComments);
     new Audio(popSound).play();
   }
-  
+
   function popwithupvote() {
     new Audio(popSound).play();
-    setDisplayedVotes(displayedVotes +1);
+    setDisplayedVotes(displayedVotes + 1);
     setNewVote(1);
+    incrementArticleVotes(article_id, newVote).then((data)=> {
+      setDisplayedVotes(updatedArticle.votes)
+    });;
   }
   function popwithdownvote() {
     new Audio(popSound).play();
-    setDisplayedVotes(displayedVotes -1);
+    setDisplayedVotes(displayedVotes - 1);
     setNewVote(-1);
+    incrementArticleVotes(article_id, newVote).then((data)=> {
+      setDisplayedVotes(updatedArticle.votes)
+    });
   }
```

Comment:
These two functions look very similar. I would say a refactor at some point would beable to make this into one reusable function and make your code DRYer 🙂